### ISSUE TITLE
Option to select destination directory for auto-accepted file transfers

### ISSUE THREAD

**Issue by cebe** [uTox/uTox#740]
currently it seems files are stored at current working directory of utox, this does not make much sense and might not even work when installed in /usr/local/bin.
There should be an option to choose that directory.

**Comment by fcore117** [comment]
Yes normal installed uTox should able to use folder selection so everyone can choose their favorite auto accept location.
